kuka_rsi_hw_interface/CMakeLists.txt

```diff
@@ -23,35 +23,44 @@ find_package(kroshu_ros2_core REQUIRED)
 find_package(hardware_interface REQUIRED)
 find_package(controller_manager_msgs REQUIRED)
 find_package(pluginlib REQUIRED)
+find_package(angles REQUIRED)
 
 find_package(tinyxml_vendor REQUIRED)
 find_package(TinyXML REQUIRED)
 
 include_directories(include ${TinyXML2_INCLUDE_DIRS})
```

Comment:
remove this

Reply:
why?

Reply:
in the new driver, tinyxml is not used (find_package is also not necessary

Reply:
I meant only the tinyxml include dirs

Reply:
I thought it was pointing to the find package angles